diktat-gradle-plugin/build.gradle.kts

```diff
@@ -0,0 +1,45 @@
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
+
+plugins {
+    `java-gradle-plugin`
+    kotlin("jvm") version "1.4.10"
+}
+
+repositories {
+    flatDir {
+        // to use snapshot diktat without necessary installing
+        dirs("../diktat-rules/target")
+    }
+    mavenCentral()
+    jcenter()
+}
+
+dependencies {
+    implementation(kotlin("gradle-plugin-api"))
+
+    implementation("com.pinterest.ktlint:ktlint-core:0.39.0") {
```

Comment:
we will need to update these versions somehow together with diktat

Reply:
I've removed explicit versions from build.gradle. They'll be synchronized with maven now.